src/test/java/teammates/test/cases/automated/InstructorCourseJoinEmailWorkerActionTest.java

```diff
@@ -37,11 +42,12 @@ public void allTests() {
         verifyNumberOfEmailsSent(action, 1);
         
         EmailWrapper email = action.getEmailSender().getEmailsSent().get(0);
+        
         assertEquals(String.format(EmailType.INSTRUCTOR_COURSE_JOIN.getSubject(), course1.getName(),
                                    course1.getId()),
                      email.getSubject());
-        assertEquals(instr1InCourse1.email, email.getRecipient());
-        
+        assertEquals(inviteReceiver.email, email.getRecipient());
+        EmailChecker.verifyEmailContainsInviterInfo(inviter, email.getContent());
```

Comment:
This is not necessary. Email generating is already comprehensively tested in `EmailGeneratorTest`, and I see that an email file for testing has been changed which is enough to show that the enhancement works as expected.